diff --git a/src/hooks/dhcp/lease_cmds/Makefile.am b/src/hooks/dhcp/lease_cmds/Makefile.am
index 2c36689..fb6365e 100644
--- a/src/hooks/dhcp/lease_cmds/Makefile.am
+++ b/src/hooks/dhcp/lease_cmds/Makefile.am
@@ -36,7 +36,7 @@ liblease_cmds_la_CXXFLAGS = $(AM_CXXFLAGS)
 liblease_cmds_la_CPPFLAGS = $(AM_CPPFLAGS)
 
 # install the shared object into $(libdir)/hooks
-lib_hooksdir = $(libdir)/hooks
+lib_hooksdir = $(libdir)/kea/hooks
 lib_hooks_LTLIBRARIES = libdhcp_lease_cmds.la
 
 libdhcp_lease_cmds_la_SOURCES  =
